lib/msf/core/post/file.rb

```diff
@@ -354,21 +354,13 @@ def rm_f(*remote_files)
   # @param old_file [String] Remote file name to move
   # @param new_file [String] The new name for the remote file
   def rename_file(old_file, new_file)
-    if session.respond_to? :commands && session.commands.include?("stdapi_fs_file_move")
+    if session.type == "meterpreter"
```

Comment:
It's probably clearer if you look at the individual commit - rename would throw an exception on any meterpreter:

https://github.com/bcook-r7/metasploit-framework/commit/f2edf21b9dd589a2cd83d8785a599afc30af8fa8

If there is a better way to resolve it, I'd be glad to change it.